how-to/join_kubeflow_ecosystem.md

```diff
@@ -0,0 +1,145 @@
+# Application for a Project to Join the Kubeflow Ecosystem
+
+Please see the [proposals/new-project-join-process.md](Documentation) to
+better understand the full process for submitting a new project.
+In short, copy this Application Template and populate the document.
+--!>
+
+## CNCF checklist
+
+- [ ] All project metadata and resources are vendor-neutral
+- [ ] Governance structure
+- [ ] Contributing guides
+- [ ] Public list of adopters.
+
+
+## Background information
+
+1. Submitter Name
+    -
+
+1. Submitter’s relationship to project / title
+    -
+
+1. Project Name
+    -
+
+1. Why is this project is valuable to the Kubeflow Community?
+    -
+
+1. Why is it beneficial for this project to be a part of the Kubeflow Community?
+    - 
+
+1. List of existing (and potential) integrations with Kubeflow Core components
+    -
+
+1. Short Description / Functionality
+    -
+
+1. Adoption
+    - 
+
+1. License Agreement
+    - 
+
+1. Vendor Neutrality
+    - 
+
+1. Trademark transition
+    - 
+
+1. CI/CD Infra Requirements
+    - 
+
+1. Governance Structure 
+    -
+
+1. Website
+    -
+
+1. GitHub repository
+    -
+
+1. 1st Release date
+    -
+
+1. Project Meeting Times
+    -
+
+1. Meeting Notes
+    -
+
+1. Installation Documentation
+    -
+
+1. Project Documentation
+    -
+
+1. Security Profile (CVE scanning, Pod Security Standards, Network Policies)
+    -
+
+1. Ownership / Legal Profile (license type, any potential issues for CNCF)
+    -
+
+1. Authorization, Isolation mechanisms
+    -
+
+1. Project Roadmap
+    -
+
+1. Other Information
+    -
+
```

Comment:
I recommend to include an item to describe if the project is already part of the CNCF ecosystem.

Reply:
Good call.